Remove version pinning for numpy, keras, tensorflow, and tensorflow-probability in pyproject.toml
Tested to obtain exact result with the one found on EESSI 2023a
diff --git a/pyproject.toml b/pyproject.toml
index ca64ce0..0ba4fe6 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -23,10 +23,10 @@ requires-python = "~=3.10"
 dependencies = [
     "click",
     "h5py",
-    "numpy==1.26.4",
-    "keras==2.10.0",
-    "tensorflow==2.10.0",
-    "tensorflow-probability==0.18.0",
+    "numpy",
+    "keras",
+    "tensorflow",
+    "tensorflow-probability",
 ]
 
 [project.urls]
